### PR TITLE
First version of chart with scaling

### PR DESCRIPTION
**Pull request by jgleespen**

### CODE REVIEW

File: app/src/main/java/com/company/strengthtracker/domain/util/DataSet.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    var yMax = Float.MIN_VALUE
+    var yMin = Float.MAX_VALUE
+
+    fun init() {
+        coordinateArray[0].forEach { x ->
+                xMax = if (x > xMax) x else xMax
+                xMin = if (x < xMin) x else xMin
+        }
+        coordinateArray[1].forEach { y ->
+            yMax = if (y > yMax) y else yMax
```
Comment (Tyler-Lopez) [inline app/src/main/java/com/company/strengthtracker/domain/util/DataSet.kt:21]: Try `maxOf(y, yMax)`

File: app/src/main/java/com/company/strengthtracker/domain/util/DataSet.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            yMin = if (y < yMin) y else yMin
```
Comment (Tyler-Lopez) [inline app/src/main/java/com/company/strengthtracker/domain/util/DataSet.kt:22]: sim but `minOf`

**Comment by jgleespen** [comment]
Bruh

File: app/src/main/java/com/company/strengthtracker/domain/util/GraphData.kt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  val yListInitial = yListInitial
+  val yListCurrent = yListCurrent
+  val totalYMaxInit = totalYMaxInit
+  val totalYMinInit = totalYMinInit
+  val totalXMax = totalXMax
+  val totalXMin = totalXMin
+  val padding = padding
+  val coordinateFormatter = coordinateFormatter
+}
+
```
Comment (Tyler-Lopez) [inline app/src/main/java/com/company/strengthtracker/domain/util/GraphData.kt:14]: you should make this a "data class" and prefix all the variables with val, you don't need any of those setter initializations

**Comment by jgleespen** [comment]
I removed this days ago idk why it's on main

**Pull request merged**